substantial/workflows/context.py

```diff
@@ -144,3 +148,21 @@ def cancel_run(self):
         Interrupt after the call
         """
         raise CancelWorkflow(self.run)
+
+
+class Utils:
+    def __init__(self, ctx: Context):
+        self.ctx = ctx
+
+    def now(self) -> str:
+        return self.ctx.save(lambda: datetime.now(tz=timezone.utc).isoformat())
```

Comment:
I suggest giving `tz` as an optional argument to `now` itself. Also returning an actual `datetime` to avoid manual conversion by the user would be better I think.

Reply:
Datetime is not serializable, that why

Reply:
Maybe parse/deserialize the saved serialized version?